src/dailies.ts

```diff
@@ -464,6 +466,24 @@ export function gin(): void {
   }
 }
 
+export function internetMemeShop(): void {
+  const baconValue = saleValue($item`BACON`);
+
+  const internetMemeShopProperties = {
+    _internetViralVideoBought: $item`viral video`,
+    _internetPlusOneBought: $item`plus one`,
+    _internetGallonOfMilkBought: $item`gallon of milk`,
+    _internetPrintScreenButtonBought: $item`print screen button`,
+    _internetDailyDungeonMalwareBought: $item`daily dungeon malware`,
+  };
+
+  for (const [property, item] of Object.entries(internetMemeShopProperties)) {
+    if (!get<boolean>(property) && baconValue * coinmasterPrice(item) < saleValue(item)) {
```

Comment:
should we make sure we have enough BACON to do the transaction, or will this acquire bacon as needed?

Reply:
acquire figures it out automatically

Reply:
no, no you were right i just had a bunch of them